spec/dummy/client/app/components/HelloWorld.jsx

```diff
@@ -10,12 +11,12 @@ class HelloWorld extends React.Component {
   }
 
   _handleChange() {
-    const name = React.findDOMNode(this.refs.name).value;
+    const name = Common.reactFindDOMNode(this.refs.name).value;
```

Comment:
So I'm abstracting the couple places there are differences in the apps. For the react_on_rails Gem, that's all hidden inside the gem.